cmd/manager/main.go

```diff
@@ -49,6 +52,18 @@ func doMain() int {
 		return 1
 	}
 
+	numOfClients := defaultNumberOfClients
+	envNumOfClients, found := os.LookupEnv(envVarBundlesSyncerNumberOfClients)
```

Comment:
None of the assigned variables (envNumOfClients, found) are used outside of the if statement's scope therefore the two lines can be combined (for best practices and consistency of code style throughout the repo).

Reply:
done